kafka/pom.xml

```diff
@@ -31,6 +31,38 @@
     <basepom.test.timeout>120</basepom.test.timeout>
   </properties>
 
+  <build>
+  <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-shade-plugin</artifactId>
+      <executions>
+        <execution>
+          <id>shade-kafka</id>
+          <phase>package</phase>
+          <goals>
+            <goal>shade</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+        <shadedArtifactAttached>false</shadedArtifactAttached>
+        <artifactSet>
+          <includes>
+            <include>org.apache.kafka</include>
+          </includes>
+        </artifactSet>
+        <relocations>
+          <relocation>
+            <pattern>org.apache.kafka</pattern>
+            <shadedPattern>otj.logging.kafka</shadedPattern>
+          </relocation>
+        </relocations>
+      </configuration>
+    </plugin>
+  </plugins>
+  </build>
+
```

Comment:
So this means we'll have to be more on top of updates for this library if we want to include, for example, bugfixes in upstream Kafka.  How will we make sure we're on top of these updates?